python/paddle/tensor/stat.py

```diff
@@ -253,6 +253,140 @@ def numel(x, name=None):
     return out
 
 
+def nanmedian(x, axis=None, ignore_nan=True, keepdim=True, name=None):
```

Comment:
there is no parameter of ignore_nan in [nanmedian RFC](https://github.com/PaddlePaddle/community/pull/89). if we want to add this parameter, should change RFC first

Reply:
This RFC doc was updated  with a new PR:  https://github.com/PaddlePaddle/community/pull/124

Reply:
I discussed this case with some paddle experts today.
In Paddle v2 3(coming this month) C + + API will be officially introduced, a program (such as model Networking) can be composed by basic C + + APIs instead of Python APIs, which is very useful for some performance pursuit scenarios.
In order to reduce the threshold for users to use C + + API, it is necessary to ensure that the experience of using C + + API is the same with Python, that is, **the parameters of the two should be the same** (the different ones are being modified in plan), and the documents of Python API can be reused without writing another document about C + + API.
Therefore, in this case, the RFC doc does not need to be modified, and the parameter `ignore_nan` should be moved to a kernel function (e.g. BaseMedianLernel), and then both nanmediankernel and mediankernel call this function using different value of `ignore_nan`. So the parameters of Python API, C + + API and kernel are the same, and the paddle Median and padding Nanmedian is used in the same way.

Reply:
Your proposal is quite reasonable. I will move the logic into c++ implementation. I will adopt this structure, BaseMedianKernel(with parameters `axis`, `keepdim`),  and then calculate the presence of nan values,  and futher determine which kernel to pick (nanmediankernel and mediankernel). This param `ignore_nan` wont be used any more.  If I understood incorrectly, pls let me know, thanks!

Reply:
done